compiler/env/FEBase.cpp

```diff
@@ -52,32 +52,6 @@ TR::FECommon::createDebug( TR::Compilation *comp)
 
 void feOutOfMemory(void *, TR::Compilation *comp, const char *reason)
```

Comment:
I don't think anything every needed the feOutOfMemory definition, it was only every called from outOfMemory. I left it as is for now because there are still a ton of calls to outOfMemory that I have to get rid of in a Part 2 PR. I'll clean that up once there are no calls to outOfMemory.